Modules/Sources/Networking/Model/SiteAPI.swift

```
@@ -42,15 +46,18 @@ public struct SiteAPI: Decodable, Equatable, GeneratedFakeable {
 
         let siteAPIContainer = try decoder.container(keyedBy: SiteAPIKeys.self)
         let namespaces = siteAPIContainer.failsafeDecodeIfPresent([String].self, forKey: .namespaces) ?? []
+        let authentication = try? siteAPIContainer.decode(Authentication.self, forKey: .authentication)
```

Question. I don't have much context: Do we want this `try?` to fail silently? Or we want to fail and throw, or log some sort of error or "auth is nil" and continue?

---

I don't want to enforce parsing this field as sites with security plugins might have this field removed/altered in some way. When parsing fails, we can assume application password is unavailable.